### ISSUE TITLE
zsh autocd not working

### ISSUE THREAD

**Issue by PicoJr** [cantino/mcfly#91]
Hello,

After loading `mcfly.zsh` the `autocd` feature of zsh stopped working.

After looking online I found a similar issue in another project: https://github.com/jenv/jenv/issues/234

For now I append `setopt autocd` at the end of `mcfly.zsh`.

I'm on Ubuntu 18.04LTS, shell is zsh via oh-my-zsh.

Best regards,

PicoJr.

**Comment by GabLeRoux** [comment]
I confirm this. typing `..` doesn't work if

```bash
if [[ -r "/usr/local/opt/mcfly/mcfly.zsh" ]]; then
  source "/usr/local/opt/mcfly/mcfly.zsh"
fi
```
is placed after `~/.zshrc`'s `source $ZSH/oh-my-zsh.sh`.

Suggested workaround works. Here's what I did:

```bash
if [[ -r "/usr/local/opt/mcfly/mcfly.zsh" ]]; then
  source "/usr/local/opt/mcfly/mcfly.zsh"
  # https://github.com/cantino/mcfly/issues/91
  setopt autocd
fi
```

**Comment by GabLeRoux** [comment]
I think this was closed by #127 

I updated `mcfly`, remove `setopt autocd` line from my config, opened a new shell
```bash
cd Application
..
```
and it worked as expected 🙌

**Comment by cantino** [comment]
Great! Thanks for verifying :)
